### ISSUE TITLE
Installing via brew doesn't work

### ISSUE THREAD

**Issue by Amparose** [a1ecbr0wn/bcd#50]
Looks like it isn't in my path as the `bcd` command doesn't work after installation. Is there another step I need to do?

M1, MacOS Ventura 13.2.1
ZSH

**Comment by a1ecbr0wn** [comment]
Yes, I think you have found a bug, if the brew install is running under ksh, it seems to be doing the bash setup to the .kshrc file, i.e. putting the following in:

```
# bookmark-cd init block                                                                                                                                             
eval "$(bookmark-cd init)" 
```

To correct this please run the following and restart your ksh shell:

```
bookmark-cd -i
```

I will get this fixed.  Thank you for taking the time to submit the issue.

**Comment by a1ecbr0wn** [comment]
If you are interested what it should have put in the `.kshrc` file (and what `bookmark-cd -i` should put in) is:

```
# bookmark-cd init block                                                                                                                                             
bookmark-cd init > ~/.bcd_ksh && . ~/.bcd_ksh  
```

**Comment by Amparose** [comment]
Oh cool! That works and this seems way more robust than the crusty/abandoned Zsh bookmarks tool I have been using. 

I believe you mean `~/.zshrc` file rather than `.kshrc`.

**Comment by a1ecbr0wn** [comment]
it is weird actually, when running the `brew install bcd`, it looks like it thinks it is running from within bash, I guess even if I run brew from zsh it is invoking bash somehow, weird.  I might have to look in to how brew works and update the docs.
